packages/docs/pages/usage/create.mdx

```diff
@@ -23,10 +18,11 @@ Orama supports the following types:
 | `string`         | A string of characters.                                                     | `'Hello world'`                                                             |
 | `number`         | A numeric value, either float or integer.                                   | `42`                                                                        |
 | `boolean`        | A boolean value.                                                            | `true`                                                                      |
+| `enum`           | An enum value.   (since version 1.3.0)                                      | `'drama'`                                                                   |
```

Comment:
I would remove "(since version 1.3.0)". We also introduced vectors from v1.2.0 but the docs should always reflect the latest release (or else, it will soon become a mess)